jerry-core/config.h

```diff
@@ -552,12 +548,8 @@
 || ((JERRY_BUILTIN_WEAKREF != 0) && (JERRY_BUILTIN_WEAKREF != 1))
 # error "Invalid value for JERRY_BUILTIN_WEAKREF macro."
 #endif
-#if !defined (JERRY_BUILTIN_PROMISE) \
-|| ((JERRY_BUILTIN_PROMISE != 0) && (JERRY_BUILTIN_PROMISE != 1))
-# error "Invalid value for JERRY_BUILTIN_PROMISE macro."
-#endif
 #if !defined (JERRY_BUILTIN_PROXY) \
-|| ((JERRY_BUILTIN_PROXY != 0) && (JERRY_BUILTIN_PROXY != 1))
+|| ((JERRY_ESNEXT != 0) && (JERRY_BUILTIN_PROXY != 1))
```

Comment:
Is this really correct? Do we want to change the proxy also?

Reply:
No, that's something i swapped by mistake, which should also explain the Proxy errors that the build option tests are throwing.